generator.go

```diff
@@ -306,6 +315,21 @@ func (st *BuildStruct) emit(g *genbase.Generator) error {
 						dest.%[1]s = str
 					}
 				`, field.Name)
+		} else if field.Tag.UnixTime {
+			if field.fieldInfo.IsTime() {
+				g.Printf(`
+						// Number Field is value between -2,147,483,647 and 2,147,483,647.
+						// but, value of zero time is -62,135,596,800.
```

Comment:
this conditions will explode with `time.Time{}Add(1 * time.Second)`.

I like below code.
How about you?

```
unixtime := src.%[1]s.Unix()
if unixtime < -2147483647 {
  unixtime = -2147483647
} else if 2147483647 < unixtime {
  unixtime = 2147483647
}
```